src/backend/expungeservice/models/charge_types/parking_ticket.py

```diff
@@ -18,8 +18,9 @@ def type_eligibility(self, disposition):
             return TypeEligibility(EligibilityStatus.INELIGIBLE, reason="Ineligible under 137.225(7)(a)")
         elif ChargeUtil.dismissed(disposition):
             return TypeEligibility(
-                EligibilityStatus.ELIGIBLE, reason="Dismissed violations are eligible under 137.225(1)(b)."
+                EligibilityStatus.INELIGIBLE,
+                reason="Dismissed parking violations are effectively ineligible at the discretion of the court.",
```

Comment:
I think we can just expand a bit here instead. Basically say everything that @michaelzhang43 mentioned. "They are eligible but because courts have been rejecting these for the past years, it will be a hassle to pass...we want you to have a smooth experience...etc"